src/helpers/images.ts

```diff
@@ -0,0 +1,6 @@
+import Config from 'react-native-config';
+
+import { images } from 'app/assets';
+
+export const getLogoSource = () =>
```

Comment:
`export const logoSource = isBeta ? images.goldWalletLogoBlackBeta : images.goldWalletLogoBlack;`

it doesn't have to be a function, the variable would have better performance as it won't have to be invoked every time the component will be rerendered